src/test/java/com/ncc/aif/NistExamplesAndValidationTest.java

```diff
@@ -22,17 +22,17 @@
     private static final boolean FORCE_DUMP = false;
 
     private static final String LDC_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/LdcAnnotations#";
-    private static final String ONTOLOGY_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/SeedlingOntology#";
-    private static final CharSource SEEDLING_ONTOLOGY = Resources.asCharSource(
-            Resources.getResource("com/ncc/aif/ontologies/SeedlingOntology"),
+    private static final String ONTOLOGY_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/LDCOntology#";
+    private static final CharSource LDC_ONTOLOGY = Resources.asCharSource(
+            Resources.getResource("com/ncc/aif/ontologies/LDCOntology"),
             StandardCharsets.UTF_8);
     private static NistTestUtils utils;
 
     @BeforeAll
     static void declutterLogging() {
         // prevent too much logging from obscuring the Turtle examples which will be printed
         ((Logger) org.slf4j.LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME)).setLevel(Level.INFO);
-        utils = new NistTestUtils(LDC_NS, ValidateAIF.create(ImmutableSet.of(SEEDLING_ONTOLOGY),
+        utils = new NistTestUtils(LDC_NS, ValidateAIF.create(ImmutableSet.of(LDC_ONTOLOGY),
```

Comment:
This should use `ValidateAIF.createForLDCOntology()` and the `SEEDLING_ONTOLOGY CharSource` removed.

Reply:
@nextcen-dgemoets - I presume you mean I should remove the `CharSource LDC_ONTOLOGY`

Reply:
Yes.  Should be able to clear out some imports as well.